docs/DeveloperGuide.md

```diff
@@ -484,16 +485,16 @@ Those without any stars are user stories that were considered but will not be im
 |  `* *`   | first-time user           | see some sample contacts already available in the app                                                                | I can try out the different features without needing to add my own data (e.g allocating people to wedding, allocating task to contacts).                            |
 |  `* *`   | careless, first-time user | reload the sample contacts into the app                                                                              | I can continue trying out different features without needing to add my own data in case I accidentally cleared the contacts.                                        |
 |  `* *`   | first-time user           | see a help message showing all the commands/feature I can use                                                        | I can try out all the different features by referring to the message.                                                                                               |
-|  `* *`   | lazy user                 | have the ability to bypass certain controlled features                                                               |                                                                                                                                                                     |
-|   `*`    | user                      | assign tasks to contacts                                                                                             | I can track which tasks have been assigned to each contact.                                                                                                         |
-|   `*`    | user                      | update the status of tasks of contacts                                                                               | I can track the status of completion of the tasks assigned to contacts.                                                                                             |
+|  `* *`   | lazy user                 | want to use command keyword shortcuts                                                                                | I can quickly execute commands without needing to type the full command word.                                                                                       |
```

Comment:
I think for second part of this, can maybe rephrase to: "can use command...". Also, I think the use case this replaces (lazy user | have the ability to bypass certain controlled features) is more referring to the force flag? So maybe we should keep that one